modules/apps/frontend-theme-locos/locos-theme/src/css/_content.scss

```diff
@@ -0,0 +1,72 @@
+
+@import "_navigation";
+@import "_breadcrumb";
+@import "_animation";
+@import "_gallery";
+
+#content {
+  z-index: 1;
+}
+
+#wrapper #main-content {
+  .portlet-title-text {
+    font-family: 'Montserrat', sans-serif;
+    font-weight: 700;
```

Comment:
I see we are using ```font-weight:``` as a numeric value.  We should create a SASS variable to define these values rather than trying to remember what each numerical value renders out to be.
Ex.
```
$normal: 400;
$bold: 600;
$bolder: 700;
```